source/layouts/layout.erb

```diff
@@ -3,15 +3,15 @@
   <head>
     <meta charset="utf-8">
     <meta content="IE=edge" http-equiv="X-UA-Compatible">
-    <% page_data = current_page.data %>
-    <% keywords = page_data.keywords ? page_data.keywords : data.seo.keywords %>
-    <% description = page_data.description ? page_data.description : data.seo.description %>
-    <% author = page_data.author ? page_data.author : data.seo.author %>
-    <title><%= page_data.title %></title>
+    <% title = "💎 Triangle Ruby Meetup 💎" %>
+    <title><%= title %></title>
```

Comment:
Hmm I'm not sure I like the `title` change. Is there a reason you prefer this to the existing `Triangle.rb | Ruby NC`?